libtock/screen.h

```diff
@@ -0,0 +1,66 @@
+/*L3GD20 3 axis gyroscope and temperature sensor
+*
+* <https://www.pololu.com/file/0J563/L3GD20.pdf>
+*
+*/
```

Comment:
This should probably be something more about the general screen interface?

Reply:
Yes, sorry for this